test/evergreen.yml

```diff
@@ -1117,6 +1117,16 @@ tasks:
             set -o verbose
             ./test/evergreen/verify_wt_datafiles.sh 2>&1
 
+  - name: clang-analyzer
```

Comment:
Depends on if WT-5176 is merged first, we need to tag this task as `pull_request`. 
Also, I'd prefer this new task being put next to other pull request tasks.

Reply:
OK, we can merge that after WT-5176.